README.md

```diff
@@ -77,6 +77,8 @@ release of Alda.
 
 [alda-install]: https://alda.io/install
 
+MacOS (UnIdentified Developer): Settings -> Privacy & Security -> 'Open Anyway'
```

Comment:
I think I'd rather include this information on the [install page](https://alda.io/install/) of the Alda website.

We could include it under the Step 3 heading, where we already have a couple of similar instructions for Windows and Linux.

Would you be interested in making a PR to the alda.io repo? You could add the information [here](https://github.com/alda-lang/alda.io/blob/68fc18ca810b6c50aaf0685faa21b724359a2e45/assets/js/install-page.js#L121).